presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -145,6 +145,8 @@
 
     private boolean legacyHiveViewTranslation;
 
+    private String viewFsTemporaryStagingDirectoryPath = "../.hive-staging";
```

Comment:
Please mention this default path change to commit body. Ideally, it would be better to separate a commit.

Reply:
Changed the commit message, thanks!

Reply:
Commit message is too long (88char) now. Let's separate into subject and body. 
```
Make ViewFS temp staging directory configurable

Additionally, changes the default path to ../.hive-staging.
```